include/Point.h

```diff
@@ -79,21 +80,17 @@ class INFLUXDB_EXPORT Point
     static inline int floatsPrecision{defaultFloatsPrecision};
 
 protected:
-    /// A value
-    std::variant<long long int, std::string, double> mValue;
-
     /// A name
     std::string mMeasurement;
 
     /// A timestamp
     std::chrono::time_point<std::chrono::system_clock> mTimestamp;
 
-    /// Tags
-    std::string mTags;
-
-    /// Fields
-    std::string mFields;
+    //// Tags    
+    std::deque<std::pair<const std::string, const std::string>> mTags;
```

Comment:
Theoretically yes, but is there any need for a map? We only ever insert at the end and then read elements one by one in the `getTags()`. Doesn't deque have better performance in such scenarios? The map may prove better in the future (to read single chosen tags values) but I don't think it would be useful right now.